integration/docker/entrypoint.sh

```diff
@@ -52,6 +52,7 @@ case ${service,,} in
     integration/docker/bin/alluxio-master.sh
     ;;
   worker)
+    export ALLUXIO_RAM_FOLDER=${ALLUXIO_RAM_FOLDER:-/dev/shm}
```

Comment:
Although this is worker specific, does it make sense to export this somewhere else?